Sources/org.dwrs.streamdeck.omnifocus.sdPlugin/manifest.json

```diff
@@ -6,27 +6,27 @@
       "States": [
         {
           "Image": "OmniFocus",
-          "FontSize": "18", 
-          "FontStyle": "Bold", 
-          "TitleAlignment": "middle", 
+          "FontSize": "18",
+          "FontStyle": "Bold",
+          "TitleAlignment": "middle",
           "TitleColor": "#ffffff"
         }
-      ], 
-      "Tooltip": "See how many unread emails you have", 
+      ],
+      "Tooltip": "See how many unread emails you have",
```

Comment:
We should change this to match the strings in the translation files

Reply:
Good spot! 😳 
Fixed.